src/drawers/rectDrawer.ts

```diff
@@ -33,7 +33,8 @@ export module _Drawer {
       return this._labelsTooWide;
     }
 
-    public drawText(data: any[], attrToProjector: AttributeToProjector, userMetadata: any, plotMetadata: Plot.PlotMetadata) {
+    public drawText(data: any[], attrToProjector: AttributeToProjector, boundingBox: D3.Selection, userMetadata: any,
+        plotMetadata: Plot.PlotMetadata) {
```

Comment:
Instead of needing to use the boundingBox here, I believe drawers have a renderArea selection that knows about the dimensions in which the drawer can render.  Can we use that instead?